waf

```diff
@@ -32,13 +32,13 @@ POSSIBILITY OF SUCH DAMAGE.
 
 import os, sys, inspect
 
-VERSION="1.8.21"
-REVISION="71bc2f1f9d6d313df7b3e05a55143f91"
-GIT="x"
+VERSION="2.0.6"
```

Comment:
To document changes better, make sure to mention waf update to 2.0.6 in the pull request description.

Reply:
I've included release notes for Linux in my `README.md`.  `waf 1.9` or better is required for Python2 or Python3.